projects/cm/src/lib/contextual-toolbar/contextual-toolbar.component.html

```diff
@@ -24,7 +24,7 @@
     <mat-menu #menu="matMenu" yPosition="below">
       <button mat-menu-item 
               *ngFor="let moreAction of moreActions" 
-              (click)="actionSelected(moreAction.name)">{{moreAction.tooltip ? moreAction.tooltip: moreAction.name}}</button>
```

Comment:
it is no longer need it?

Reply:
See 
![image](https://user-images.githubusercontent.com/15807748/144232722-938cf743-8864-47f6-a972-35cbf497735f.png)

Instead of using only the Action Name I am passing as Parameter the full Object
![image](https://user-images.githubusercontent.com/15807748/144232951-926164a0-fea1-448b-99d8-9190df2a0ed4.png)

Reply:
fine!!! thanks for sharing it!!! I will close it